### PR TITLE
Address deprecations in Twig 1.12

### PR DESCRIPTION
**Pull request by martin-georgiev**

### CODE REVIEW

File: src/Extension/UrlDecodeExtension.php [hunk 1]
```diff
@@ -31,7 +31,7 @@ public function urlDecode($url)
      *
      * @return string The extension name
      */
-    public function getName()
+    public function getName(): string
```
Comment (ben-challis) [inline src/Extension/UrlDecodeExtension.php]: this override is deprecated and can be dropped entirely.

File: src/Extension/YamlDumpExtension.php [hunk 2]
```diff
@@ -36,7 +36,7 @@ public function yamlDump(array $data, $level = 0)
      *
      * @return string The extension name
      */
-    public function getName()
+    public function getName(): string
```
Comment (ben-challis) [inline src/Extension/YamlDumpExtension.php]: same deal here, should be dropped entirely.

File: tests/Functional/TwigControllerTest.php [hunk 3]
```diff
@@ -0,0 +1,188 @@
...
+                                ],
+                            ],
+                        ],
+                    ],
+                ],
+            ],
+            $container
+        );
+
+        $doctrineExtensionsExtension = new LendableDoctrineExtensionsExtension();
```
Comment (ben-challis) [inline tests/Functional/TwigControllerTest.php]: I presume this is a bad copy-paste?

File: src/Extension/UrlDecodeExtension.php (hunk 1 above)
Comment (martin-georgiev) [reply src/Extension/UrlDecodeExtension.php]: good spot

File: tests/Functional/TwigControllerTest.php (hunk 3 above)
Comment (martin-georgiev) [reply tests/Functional/TwigControllerTest.php]: Indeed it was, it was corrected in one of the commits in PR #1

**Pull request merged**